src/test/java/duckling/ResponseHeaderTest.java

```diff
@@ -0,0 +1,58 @@
+package duckling;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+public class ResponseHeaderTest {
+    @Test
+    public void hasDefaultResponse() throws Exception {
+        ResponseHeaders responseHeaders = new ResponseHeaders();
+        String[] expectation = {
+            "HTTP/1.0 200 OK" + Server.CLRF,
+            "Content-Type: null" + Server.CLRF,
+            Server.CLRF
+        };
+        assertEquals(expectation, responseHeaders.toArray());
```

Comment:
Also, in case you're unaware, jUnit comes with a library called Hamcrest that contains a richer set of expressive matchers.  You access those through the `org.junit.Assert.assertThat` method and the static methods on the  `org.hamcrest.CoreMatchers` class.  My typical jUnit imports for tests look like this:

``` java
import static org.junit.Assert.assertThat;
import static org.hamcrest.CoreMatchers.*;
```